dashboard/test/lib/announcements_test_data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "pages": {
+    "/home": "teacher-apps-closing-2020",
+    "/courses": "teacher-apps-open-2020",
+    "/another-page": "not-a-banner"
+  },
+  "banners": {
+    "teacher-apps-open-2020": {
+      "image": "https://code.org/images/professional-learning-2019-3.png",
```

Comment:
I should have caught this before, but we want to avoid relying on images that are in production, which hardcoding to https://code.org here does.  That's because we are often adding the new image at the same time as we are authoring the new announcement, so the image won't be on production yet.  There are a few strategies we can use, such as using the `pegasus()` function in JS or the `CDO.code_org_url` function in dashboard, but my preferred might be that we put such images in `/shared` and refer to them like that.  That way, the same image is available at `/shared/images` on either dashboard or pegasus domains, we still get Pegasus' handy `fit-/fill-` resizing, and by referring to them relatively, we will be looking at an image on the local environment, rather than on production.